### ISSUE TITLE
[a11y-defect-3]: Use of deprecated "wizard" component results in H1 issues and increased cognitive load

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#64218]
### Point of contact

Jamie Klenetsky Fay (@coforma-jamie)

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

A [wizard pattern](https://design.va.gov/patterns/wizards), which is deprecated, is used to lead Vets to [Form 28-1900](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/):
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/86962741-9491-4de8-b146-5ff531e36897)

Using this pattern can be confusing for users. In this case, when navigating between the wizard version and intro version of the page, the `<H1>` is the same, and it may not be clear to screen reader users if they are actually navigating to separate pages.

This also doesn't fit the design system's [sub-task pattern](https://design.va.gov/patterns/help-users-to/complete-a-sub-task) model, which gets especially confusing when the user triggers the VR&E Orientation slides.

### Reproduction steps

1. Go to [this page](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)
2. Answer the questions in the wizard (Veteran, Yes, Yes). Notice that the "VR&E Orientation" slides now appear below the wizard.
3. Click on "Apply for Veteran Readiness and Employment with VA Form 28-1900"
4. Scroll to the bottom and click "go back and answer the questions again"
5. This will take you back to the wizard, which has the same title as the intro page.

### Short-term fix
You can use focus management to automatically focus on the `<H2>`, "Is this the form I need?" heading. That should make it more clear to an AT user what the page is for. Look at the [File a Supplemental Claim form](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/start) to see it in action. When the page loads, focus automatically goes to the `<H2>`:
![Screenshot 2023-08-15 at 3 22 27 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/f5ad5d51-5b18-42ff-be95-a75aa1904f84)

### Long-term solution
As mentioned in the Design System [wizard pattern](https://design.va.gov/patterns/wizards) page, the [Wizard pattern is now deprecated](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/399) and replaced by the the [help users complete a sub-task pattern](https://design.va.gov/patterns/help-users-to/complete-a-sub-task). 

There are many accessibility defects related to the wizard pattern, as described [here](https://design.va.gov/patterns/wizards#retrofit-this-pattern).

We highly recommend that you use the sub-task pattern instead of the wizard pattern to avoid this accessibility issue and reduce the user's cognitive load. This includes putting the VR&E Orientation slideshow on its own page.

### References, articles, or WCAG support

1. [Focus management example](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/start)
2. Reduce cognitive load by having [one thing per page](https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page)
3. [How to retrofit the wizard pattern, if you absolutely must keep it](https://design.va.gov/patterns/wizards#research-and-accessibility-rationale)

### Type of issue

- [ ] Axe-core
- [X] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [X] Focus
- [X] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [X] Overall user experience
- [X] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else
```[tasklist]
### Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/68008
```
